nvssclient/Controllers/RecordController.cs

```diff
@@ -155,12 +160,88 @@ public async Task<ActionResult> VoidRecordHandler([FromBody] List<object> textLi
         // Wraps the record in a FHIR Void message and queues the message to be sent to the NVSS API Server
         [HttpPost]
         [Route("void")]
-        public async Task<ActionResult> VoidRecordHandler([FromBody] object recordJson)
+        public async Task<ActionResult> VoidRecordHandler([FromBody] Dictionary<string,object> json)
         {               
             try {
                 // Create a submission message for the record
+                object recordJson = json["record"];
+                uint? blockCount = UInt32.Parse(json["block_count"].ToString());
                 DeathRecord record = new DeathRecord(recordJson.ToString(), true);
+                
                 var message = new DeathRecordVoidMessage(record);
+                message.BlockCount = blockCount;
+                InsertMessageItem(message);
+            } catch (Exception e){
+                Console.WriteLine("Error Handling Record: {0}", e);
+                return BadRequest();
+            }
+            // return HTTP status code 204 (No Content)
+            return NoContent();
+        }
+
+        // POST: Alias Records
+        // Wraps each record in a FHIR Alias message and queues the message to be sent to the NVSS API Server
+        [HttpPost]
+        [Route("aliases")]
+        public async Task<ActionResult> AliasRecordHandler([FromBody] List<object> textList)
```

Comment:
This method is declared as asynchronous but does not await anything. Either drop the `async` or use `await` to wait on an asynchronous operation (such as the database).